src/index.js

```diff
@@ -0,0 +1,6 @@
+import styles from './assets/scss/app.scss';
```

Comment:
since `styles` are not used anywhere:
```suggestion
import './assets/scss/app.scss';
```